neuralmonkey/dataset.py

```diff
@@ -14,6 +14,7 @@
 
 SERIES_SOURCE = re.compile("s_([^_]*)$")
 SERIES_OUTPUT = re.compile("s_(.*)_out")
+PREPROCESSED_SERIES = re.compile("pre_([^_]*)$")
```

Comment:
Jo, postřehnul jsem `post-edit.ini` pořádně až teď. Jo, to je asi dobrý.. Tj. pre_cokoliv je callable, ze který se vytvoří series "cokoliv", nebo rovnou series? (Viz muj review u post-edit.ini)